internal/api/common.go

```diff
@@ -70,6 +70,8 @@ const (
 	DescDir      = "desc"
 )
 
+var errAddBootstrap = errors.New("failed to add bootstrap configuration")
```

Comment:
Okay @arvindh123 . Although I think `errAddBootstrap` is a better option since `ErrCreateEntity` does not fully indicate the nature of the error and may be confusing to the user. What do you think about this?

Reply:
It look odd me, The only un exported variable in that file.  And we are duplicating the same variable in two files

Reply:
The error was un exported in the bootstrap service, let me place it in `svcerr` package so that we will not have this odd looking error.